src/st/dai/dai.h

```diff
@@ -39,20 +18,23 @@ enum OVL_EXPORT(Palette) {
     PAL_BACKGROUND_SKY_LAND = 0xC5,
     PAL_CASTLE_WALL_2 = 0xC8,
     PAL_CONFESSIONAL_GHOST = 0xC9,
-    PAL_801809C8 = 0xCA,
+    PAL_UNUSED = 0xCA, // Only exists in g_EInitUnused801809C8 definition
     PAL_CONFESSIONAL_CLUT = 0xCC,
     PAL_CONFESSIONAL_GHOST_BAD = 0xCE,
-    PAL_COLOR_GRADIENT = 0x15F,
+    PAL_WHITE = 0x15F,
     PAL_SPIKES_DUST = 0x161,
     PAL_SPECTRAL_SWORD_WEAPON_UNK_16B = 0x16B,
-    PAL_STAGE_NAME_19D = 0x19D,
-    PAL_STAGE_NAME_19E = 0x19E,
-    PAL_STAGE_NAME_19F = 0x19F,
+    PAL_LIGHT_GREY_TO_BLACK = 0x19D,
+    PAL_BLACK_TO_LIGHT_GREY = 0x19E,
+    PAL_BLACK_TO_WHITE = 0x19F,
+    PAL_CUTSCENE_UNK_4_CLUT = 0x1A1,
```

Comment:
I'd like to hold this PR for just a little bit while I validate whether these palettes belong here or in game.h.